pkg/repositories/models/execution.go

```diff
@@ -60,4 +62,11 @@ type Execution struct {
 	State *int32 `gorm:"index;default:0"`
 	// The resource type of the entity used to launch the execution, one of 'launch_plan' or 'task'
 	LaunchEntity string
+	// Tags associated with the execution
+	Tags []AdminTag `gorm:"many2many:execution_admin_tags;"`
```

Comment:
can you add the full schema of the new tables to the PR description?

Reply:
updated it

Reply:
Can you add the example schemas containing the foreign keys not the go models?

Reply:
Sorry, could you say more? What else should I add to the table schema in the PR description

Reply:
oh, I meant the actual schema of the database tables created. I see that the description mentions foreign keys and primary keys, so adding the schemas is not really necessary. Thanks!